gobblin-temporal/src/main/java/org/apache/gobblin/temporal/joblauncher/GobblinJobLauncher.java

```diff
@@ -146,23 +152,17 @@ protected void runWorkUnits(List<WorkUnit> workUnits) throws Exception {
       // Start the output TaskState collector service
       this.taskStateCollectorService.startAsync().awaitRunning();
 
-      TimingEvent jobSubmissionTimer =
-          this.eventSubmitter.getTimingEvent(TimingEvent.RunJobTimings.HELIX_JOB_SUBMISSION);
```

Comment:
The metrics are no longer relevant because they're related to Helix job submissions and timings (For Fast Ingest). These metrics don't actually log anything but they will be emitted to the GTE topic